internal/control_plane/server/services.go

```diff
@@ -31,14 +35,15 @@ func (s *clientCPServicesServer) GetAllMetadata(ctx context.Context, request *pr
 }
 
 func (s *clientCPServicesServer) GetStreamLogs(request *protobuf.RequestForStreamLog, stream protobuf.ClientCPServices_GetStreamLogsServer) error {
-	logString := &protobuf.Log{Log: "lorem ipsum logger logger logger dumb"}
+	uid := id_generator.NextID()
+	logString := &protobuf.Log{RequestId: uid, Log: "lorem ipsum logger logger logger dumb"}
 	err := stream.Send(logString)
-	logger.Error(err, "GetStream Request Received - Sending stream to client")
+	logger.Error(err, fmt.Sprintf("%v GetStream Request Received - Sending stream to client", uid))
```

Comment:
no error checking? please check the error and only log/return if it's only `err != nil`. this will always log the error even the error is nil.

Reply:
In Zerolog.go, we are using .Err(err).Msg(msg) with Logger, that logs error if error is nil, otherwise logs the message as info.

Reply:
that's not a good design. please don't create magic like that, the practice is to check the error as near as possible to the cause. please use log.Info to do log the info and `log.Error` to log the error don't log the info in `log.Error` if error is nil.